backend/packages/Upgrade/src/api/repositories/AnalyticsRepository.ts

```diff
@@ -418,100 +411,110 @@ export class AnalyticsRepository {
   }
 
   public async getCSVDataForSimpleExport(
+    experimentsData: any,
     experimentId: string,
+    isSRSExperiment: boolean,
```

Comment:
This can be determined from the experiment data itself. You don't need this parameter.

Reply:
Earlier did that, but thought of providing a separate parameter, else would have to get from the first array element of experimentData. Ok, will remove this.